src/index.js

```diff
@@ -4,297 +4,285 @@
-    } else if (device.ipod()) {
-      addClass("ios ipod mobile");
-    }
-  } else if (device.android()) {
-    if (device.androidTablet()) {
-      addClass("android tablet");
-    } else {
-      addClass("android mobile");
-    }
-  } else if (device.blackberry()) {
-    if (device.blackberryTablet()) {
-      addClass("blackberry tablet");
-    } else {
-      addClass("blackberry mobile");
-    }
-  } else if (device.windows()) {
-    if (device.windowsTablet()) {
-      addClass("windows tablet");
-    } else if (device.windowsPhone()) {
-      addClass("windows mobile");
-    } else {
-      addClass("desktop");
-    }
-  } else if (device.fxos()) {
-    if (device.fxosTablet()) {
-      addClass("fxos tablet");
-    } else {
-      addClass("fxos mobile");
-    }
-  } else if (device.meego()) {
-    addClass("meego mobile");
-  } else if (device.nodeWebkit()) {
-    addClass("node-webkit");
-  } else if (device.television()) {
-    addClass("television");
-  } else if (device.desktop()) {
-    addClass("desktop");
+if (device.ios()) {
+  if (device.ipad()) {
+    addClass("ios ipad tablet");
+  } else if (device.iphone()) {
+    addClass("ios iphone mobile");
+  } else if (device.ipod()) {
+    addClass("ios ipod mobile");
   }
-
-  if (device.cordova()) {
-    addClass("cordova");
+} else if (device.android()) {
+  if (device.androidTablet()) {
+    addClass("android tablet");
+  } else {
+    addClass("android mobile");
   }
-
-  // Orientation Handling
-  // --------------------
-
-  // Handle device orientation changes.
-  handleOrientation = function () {
-    if (device.landscape()) {
-      removeClass("portrait");
-      addClass("landscape");
-    } else {
-      removeClass("landscape");
-      addClass("portrait");
-    }
-  };
-
-  // Detect whether device supports orientationchange event,
-  // otherwise fall back to the resize event.
-  if (Object.prototype.hasOwnProperty.call(window, "onorientationchange")) {
-    orientationEvent = "orientationchange";
+} else if (device.blackberry()) {
+  if (device.blackberryTablet()) {
+    addClass("blackberry tablet");
   } else {
-    orientationEvent = "resize";
+    addClass("blackberry mobile");
   }
-
-  // Listen for changes in orientation.
-  if (window.addEventListener) {
-    window.addEventListener(orientationEvent, handleOrientation, false);
-  } else if (window.attachEvent) {
-    window.attachEvent(orientationEvent, handleOrientation);
+} else if (device.windows()) {
+  if (device.windowsTablet()) {
+    addClass("windows tablet");
+  } else if (device.windowsPhone()) {
+    addClass("windows mobile");
   } else {
-    window[orientationEvent] = handleOrientation;
+    addClass("desktop");
   }
-
-  handleOrientation();
-
-  if (typeof define === 'function' && typeof define.amd === 'object' && define.amd) {
-    define(function() {
-      return device;
-    });
-  } else if (typeof module !== 'undefined' && module.exports) {
-    module.exports = device;
```

Comment:
This is the "old" way of doing universal exports, removed in favor of letting the `nwb` conf handle via 

``` js
export default device;
```